test_tools/sigma_verify_rules.py

```diff
@@ -0,0 +1,170 @@
+# Copyright 2020 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""A tool to test sigma rules.
+This tool can be used to verify your rules before running an analyzer.
+It also does not require you to have a full blown Timesketch instance up and running.
+Example way of running the tool:
+  $ sigma_verify_rules.py --config_file ../data/sigma_config.yaml ../data/linux/
+"""
+
+
+import logging
+import os
+import argparse
+import sys
+
+
+
+from sigma.backends import elasticsearch as sigma_elasticsearch
+import sigma.configuration as sigma_configuration
+from sigma.parser import collection as sigma_collection
+logging.basicConfig(level=os.environ.get("LOGLEVEL", "ERROR"))
+import sigma.parser.exceptions
+
+
+def get_codepath():
+    """Return the absolute path to where the tool is run from."""
+    path = __file__
+    if path.startswith(os.path.sep):
+        return path
+
+    dirname = os.path.dirname(path)
+    for sys_path in sys.path:
+        if sys_path.endswith(dirname):
+            return sys_path
+    return dirname
+
+
+def run_verifier(rules_path, config_file_path):
+    """Run an sigma parsing test on a given dir and returns results from the run.
+    Args:
+        rules_path: the path to the rules.
+        config_file_path: the path to a config file that contains mapping data.
+    Raises:
+        IOError: if the path to either test or analyzer file does not exist
+                 or if the analyzer module or class cannot be loaded.
+    Returns:
+        two lists:
```

Comment:
not quite right... you return a Tuple with two items in it, each of which is a list... so uyou return a tuple of two lists

Reply:
done